monitoring/snippets/v3/uptime-check-client/noxfile_config.py

```diff
@@ -22,21 +22,18 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    'ignored_versions': ['2.7'],
-
+    "ignored_versions": ["2.7"],
     # Declare optional test sessions you want to opt-in. Currently we
```

Comment:
If you're changing the noxfile_config then I suggest adding typehints enforcing, part of the template shows:
```
    # Old samples are opted out of enforcing Python type hints
    # All new samples should feature them
    "enforce_type_hints": True,
```

Reply:
I did not change it. It was changed by `black .` command